### PR TITLE
*: partial staticcheck fix

### PR DESCRIPTION
**Pull request by wpedrak**
Fixes to part of `staticcheck` complains as step to potentially incorporate static checker into regular testing procedure.

### CODE REVIEW

File: raft/rafttest/node.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func startNode(id uint64, peers []raft.Peer, iface iface) *node {
 
 func (n *node) start() {
 	n.stopc = make(chan struct{})
```
Comment (ptabor) [inline raft/rafttest/node.go:66]: without C here.

File: raft/rafttest/node.go [hunk 2]
```diff
@@ -64,7 +64,7 @@ func startNode(id uint64, peers []raft.Peer, iface iface) *node {
...
-	ticker := time.Tick(5 * time.Millisecond)
+	ticker := time.NewTicker(5 * time.Millisecond).C
```
Comment (ptabor) [inline raft/rafttest/node.go:68]: defer ticker.Close()

File: raft/rafttest/node.go (hunk 1 above)
Comment (ptabor) [reply raft/rafttest/node.go:66]: <-ticker.C below. (line 72)

**Review by ptabor — APPROVED** [review]
Thank you.

File: raft/rafttest/node.go (hunk 2 above)
Comment (wpedrak) [reply raft/rafttest/node.go:68]: I bet you meant `ticker.Stop()`

Comment (ptabor) [reply raft/rafttest/node.go:68]: yes

**Pull request merged**